app/routes/docs.circulars.corrections/route.mdx

```diff
@@ -0,0 +1,54 @@
+---
+handle:
+  breadcrumb: Corrections
+---
+
+import requestbuttonscreenshot from './request_correction_button.png'
+import versionbuttonscreenshot from './version_button.png'
+
+# Corrections
+
+Circulars authors can request edits to their own archived Circulars to correct inaccurate information, especially when the correction is important for the follow-up community's activities. These may include author lists, subject corrections, typos, and incorrect citations. This capability should not be used to add details of new observations or analyses — instead users should submit new Circulars with appropriate subjects (e.g. "refined analysis" or "additional observations").
+
+## Who can request corrections?
+
+Anyone with a [peer endorsement](/docs/circulars/submitting#to-request-a-peer-endorsement) can submit requests for corrections. If the requester is the submitting author, the moderator will likely approve the request immediately. If the requester is a co-author or community member, the moderator will get concurrence from the submitting author via email.
+
+## How to request corrections?
+
+- [Sign in to your GCN account](/login)
+- Navigate to the [archive page](/circulars) for that particular circular.
+- Select "Request Correction"
```

Comment:
Can you move the image to below this bullet, and add something like a rectangle around the button to make it clear that this is an image referencing the page and not form buttons? The screenshot blends in with the current page and looks like it is clickable: 
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/f8c57df5-fab1-43be-aa7d-15c7a4f8ebaf)